src/nedap/components/pedestal/service/component.clj

```diff
@@ -9,17 +9,26 @@
    [nedap.speced.def :as speced]
    [nedap.utils.modular.api :refer [implement]]))
 
+(speced/def-with-doc ::pedestal.http/join?
+  "A true value blocks the thread until server ends.
+  
+  Warning: Blocking the server thread (true value)can interfere with shutdown
+  hooks that would need access to the started component/system.
+  
+  Default to false (non-blocking)."
```

Comment:
* blocks the **calling** thread
* component/system -> Component System
* Surround `true` with backticks
* Remove `Default to false (non-blocking)`, it duplicates info